requirements.txt

```diff
@@ -27,8 +26,6 @@ pexpect==4.1.0
 pathlib2==2.1.0
 pickleshare==0.7.2
 simplegeneric==0.8.1
-path.py==8.2.1
-backports.shutil_get_terminal_size==1.0.0
```

Comment:
I think they may need to be re-packaged, not just renamed: `AssertionError: path-py==8.2.1 .dist-info directory not found`

I haven't read much about pip's issue with dots and underscores... I'll find out what solution they propose for packages like this.. the backports packages all have dots in their names.